compose/compose.go

```diff
@@ -307,7 +316,12 @@ func getAlpha(idx int, raft string) service {
 		maxZeros = opts.NumZeros
 	}
 
-	zeroHostAddr := fmt.Sprintf("%s:%d", getHost("zero1"), zeroBasePort+opts.PortOffset)
+	zeroName := "zero"
+	if opts.ContainerPrefix != "" {
```

Comment:
`getHost()` logic somehow got removed? Don't we need that?

Reply:
updated